code/websites_and_search_engines.py

```diff
@@ -11,6 +11,11 @@
 )
 
 website_defaults = {
+    "talon website":"http://talonvoice.com",
+    "talon slack":"http://talonvoice.slack.com/messages/help",
```

Comment:
let's switch these to `https`
```suggestion
    "talon website":"https://talonvoice.com",
    "talon slack":"https://talonvoice.slack.com/messages/help",
```

Reply:
@rntz / @tararoys looks like maybe you missed this comment?